check_import.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python3
```

Comment:
@vyskocilpavel We need to decide the final version of the sync output and how the probe should respond.